app/components/lists/TabButton.js

```diff
@@ -22,7 +13,7 @@ const TAB_BUTTONS = [
   {
     id: "detail-tab",
     href: "#detail",
-    label: "Outras informações",
+    label: "Dicas",
```

Comment:
start using enums for these types of situation (later you check if label == something)

https://www.codecademy.com/resources/docs/javascript/enums

https://masteringjs.io/tutorials/fundamentals/enum